### ISSUE TITLE
1/2 of tests failing unexpectedly because of Uncaught RuntimeError

### ISSUE THREAD

**Issue by jackzampolin** [webdriverio/gulp-webdriver#11]
Every time I run my tests the first half run correctly while the second half all fail with the same error:

``` bash
     Uncaught RuntimeError
     (UnknownError:13) An unknown server-side error occurred while processing the command.
     Problem: POST /session//url
Build info: version: '2.46.0', revision: '87c69e2', time: '2015-06-04 16:16:47'
System info: host: 'eruditorum.att.net', ip: '192.168.1.222', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.10.4', java.version: '1.8.0_51'
Driver info: driver.version: unknown
```

Looking around online I found a [few people](https://github.com/detro/ghostdriver/issues/140) [having similar issues](http://stackoverflow.com/questions/26492793/selenium-ends-randomly-with-uncaught-error) but no fixes for running the tests through gulp.  Any advice here or am I fucking something up?

**Comment by christian-bromann** [comment]
Do you still have the same issue with the new released version 1.0?

**Comment by christian-bromann** [comment]
Closing due to inactivity
